File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        z=x
        r=0
        while x>0:
            y=x%10
            x=(x-y)/10
            r=r*10+y
        return r==z
            
    
        """
        :type x: int
        :rtype: bool
        """
        

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int z=x;
        long r=0;
        int y;
        if (x<0){
            return false;
        }
        while (z){
            y=z%10;
            r=r*10+y;
            z=z/10;
        }
        return r==x;        
    }
};